apps/backend/src/app/app.module.ts

```diff
@@ -22,13 +24,14 @@ import { FirebaseApp } from './firebase/firebase-app';
         username: configService.get<string>('DATABASE_USER'),
         password: configService.get<string>('DATABASE_PASSWORD'),
         database: configService.get<string>('DATABASE_NAME'),
-        entities: [User],
+        entities: [User, Transaction],
         synchronize: true,
       }),
       inject: [ConfigService],
     }),
     UserModule,
-    TypeOrmModule.forFeature([User]),
+    TransactionModule,
+    TypeOrmModule.forFeature([User, Transaction]),
```

Comment:
Remove from app module

Reply:
without this line get an error
![image](https://github.com/user-attachments/assets/432274a3-0b0a-46b3-8d4c-e9ab400fdd93)